### DISCUSSION TITLE
[notification] There is a option in MessageService to set timeout without action?

### DISCUSSION THREAD

**Discussion by eli88p** [eclipse-theia/theia#7863] [General] [answered]
Hi,

I want to show notification via @theia/MessageService.
i can't find the way to show message with timeout but without action field

Thanks for help
Eli.

**Comment by vince-fugnitto** [comment, marked as answer]
@eli88p thank you for the question, I believe there is an issue tracking the bug (timeout without any actions): https://github.com/eclipse-theia/theia/issues/6707
